packages/ozone/src/db/migrations/20240710T051104136Z-sets.ts

```diff
@@ -0,0 +1,63 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  // Create the sets table
+  await db.schema
+    .createTable('ozone_set')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+    .addColumn('name', 'varchar(128)', (col) => col.notNull().unique())
+    .addColumn('description', 'varchar(1024)')
```

Comment:
IMO it's probably fine to just use `varchar` if you like.  In postgres there's no difference in the storage between `varchar(128)`, `varchar`, and `text` under the hood.  It just adds a validation check on the length.